runtime/jvmTest/src/kotlinx/serialization/features/SerializerByTypeTest.kt

```diff
@@ -183,4 +183,12 @@ class SerializerByTypeTest {
         val superType = base::class.java.genericSuperclass!!
         return (superType as ParameterizedType).actualTypeArguments.first()!!
     }
+
+    enum class Foo
+
+    @Test
+    fun testNonSerializableEnum() {
+        val serializer = serializer<Foo>()
+        assertTrue { serializer.descriptor.kind is SerialKind.ENUM }
```

Comment:
```suggestion
        assertTrue(serializer.descriptor.kind is SerialKind.ENUM)
```

no need for lambda, actually

Reply:
Not sure what's the difference here, but okay